test/plugins/features/movies/controller.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie = require('../../../../lib/models/movie');
+
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it ('creates a movie', () => {
+      const payload = { title: 'WALL-E'};
```

Comment:
Nit: One-line object literals should be padded by spaces, so something like this:
```js
{ title: 'WALL-E' }
```